src/runmode-unix-socket.c

```diff
@@ -50,13 +50,14 @@
 #include "conf-yaml-loader.h"
 
 #include "datasets.h"
+#include <stdint.h>
```

Comment:
Not needed -- comes from including `suricata-common.h

Reply:
Updated now. Apologies for missing it in this file when removing it in the other file(s). Thanks